src/bulkOperationCommand.ts

```diff
@@ -93,11 +98,26 @@ export abstract class BulkOperationCommand extends BulkBaseCommand {
     return job.check();
   }
 
+  private printBulkErrors(failedResults: IngestJobV2FailedResults<Schema>): void {
+    const columns = {
+      id: { header: 'Id' },
+      error: { header: 'Error' },
+    };
+    const options = { title: `Bulk Failures [${failedResults.length}]` };
+    ux.log();
```

Comment:
you can create a "json aware ux" with `const ux = new Ux({jsonEnabled: this.jsonEnabled()});` You might need to move this so it has access to the command's `json` flag

Reply:
I can change this, of course. But since the Ux should never be printed if `this.jsonEnabled()` is `true` (because the error results are already part of the returned JSON itself), I think we can assume that we'd really like to print a human readable table here?